x-pack/plugin/spatial/src/main/java/org/elasticsearch/xpack/spatial/index/mapper/GeoShapeWithDocValuesFieldMapper.java

```diff
@@ -344,4 +341,114 @@ protected void checkIncomingMergeType(FieldMapper mergeWith) {
         }
         super.checkIncomingMergeType(mergeWith);
     }
+
+    public static class GeoShapeDocValuesField
+        implements
+            DocValuesField<GeoShapeValues.GeoShapeValue>,
```

Comment:
Is this formatting intended?

Reply:
Guess so since this is what spotless produced.